src/account/UpgradeableModularAccount.sol

```diff
@@ -137,35 +136,30 @@ contract UpgradeableModularAccount is
         return execReturnData;
     }
 
-    /// @notice Executes a transaction from the account
-    /// @param execution The execution to perform
-    /// @return result The result of the execution
-    function execute(Execution calldata execution)
+    /// @inheritdoc IStandardExecutor
+    function execute(address target, uint256 value, bytes calldata data)
         external
         payable
         override
         wrapNativeFunction
         returns (bytes memory result)
     {
-        result = _exec(execution.target, execution.value, execution.data);
+        result = _exec(target, value, data);
     }
 
-    /// @notice Executes a batch of transactions from the account
-    /// @dev If any of the transactions revert, the entire batch reverts
-    /// @param executions The executions to perform
-    /// @return results The results of the executions
-    function executeBatch(Execution[] calldata executions)
+    /// @inheritdoc IStandardExecutor
```

Comment:
Can we adopt this doc format for `IPluginExecutor`?

Reply:
Updated